### DISCUSSION TITLE
FirebaseError: Expected first argument to collection() to be a CollectionReference, a DocumentReference or FirebaseFirestore, js engine: hermes

### DISCUSSION THREAD

**Discussion by kyeltech** [invertase/react-native-firebase#6768] [Q&A]
apparently, i'm trying to use firebase for a small school project, also my first time using firebase after learning about it years ago, I'll appreciated any quick response

**Comment by mikehardy** [comment]
Hi there! with apologies, this is not an actionable discussion starter.
https://stackoverflow.com/help/how-to-ask
https://stackoverflow.com/help/minimal-reproducible-example
